tutorials/create-cloud-build-image-factory-using-packer/index.md

```diff
@@ -41,59 +59,297 @@ gcloud config set compute/zone $ZONE
 gcloud config set account $ACCOUNT
 ```
 
-## Set the project variable
+</details>
+
+## Building a Windows Image
+
+<details>
+<summary>Click to reveal commands</summary>
+
+```powershell
+$env:PROJECT="NEW PROJECT NAME"
+$env:ORG="YOUR ORGANIZATION NAME"
+$env:BILLING_ACCOUNT="YOUR_BILLING_ACCOUNT_NAME"
+$env:ZONE="COMPUTE ZONE YOU WANT TO USE"
+$env:ACCOUNT="GOOGLE ACCOUNT YOU WANT TO USE" or $(gcloud config get-value account)
+
+gcloud projects create "$env:PROJECT" --organization=$(gcloud organizations list --format="value(name)" --filter="(displayName='$env:ORG')")
+gcloud beta billing projects link $env:PROJECT --billing-account=$(gcloud alpha billing accounts list --format='value(name)' --filter="(displayName='$env:BILLING_ACCOUNT')")
+gcloud config configurations create --activate $env:PROJECT
+gcloud config set project $env:PROJECT
+gcloud config set compute/zone $env:ZONE
+gcloud config set account $env:ACCOUNT
+```
+
+</details>
+
+# Task 1b: Set the project variable
 
 Skip this step if you created a new project in the previous section.
 
 Ensure that you're working with the project that you want to use with `gcloud`.
 For more information on configurations see [configurations](https://cloud.google.com/sdk/gcloud/reference/config/configurations/).
 Fill in `[CONFIGURATION NAME]` with the name of the configuration you want to use.
 
+## Building a Linux Image
+
+<details>
+<summary>Click to reveal commands</summary>
+
 ```sh
 gcloud config configurations activate [CONFIGURATION NAME] #The configuration for the project you want to use
 PROJECT=$(gcloud config get-value project)
 ```
 
-## Copy the files for this tutorial to a new working directory and git repository
+</details>
+
+## Building a Windows Image
+
+<details>
+<summary>Click to reveal commands</summary>
+
+```powershell
+gcloud config configurations activate [CONFIGURATION NAME] #The configuration for the project you want to use
```

Comment:
Not necessary?

Reply:
I agree that the hiding of the commands behind the details element isn't necessary and is even problematic. Readers often fail to see and use such expandable sections well. Please remove the details and summary tags.

Reply:
My thoughts on using the details/expandable section format:

- Better distinguish between Linux VM vs Windows VM tutorial
- Avoid potential scenarios where users mix up commands between Linux vs Windows tutorials ie. from following instructions top down instead of every second section to follow respective OS instructions
- Cleaner and shorter markdown vs having everything in one long post 

Let me know you're thoughts